Sources/Extensions/Watch/ComplicationController.swift

```diff
@@ -41,7 +51,16 @@ class ComplicationController: NSObject, CLKComplicationDataSource {
         for complication: CLKComplication,
         withHandler handler: @escaping (CLKComplicationPrivacyBehavior) -> Void
     ) {
-        handler(.showOnLockScreen)
+
+        let model: WatchComplication?
+
+        model = getComplicationModel(for: complication)
```

Comment:
if you pull this up into the variable declaration above you can avoid needing to declare the type, too